ipatests/test_integration/test_installation.py

```diff
@@ -567,3 +567,51 @@ def test_KRA_install_after_cert_renew(self):
         self.master.run_command(['kinit', 'admin'], stdin_text=passwd)
         cmd = self.master.run_command(['ipa-kra-install', '-p', dm_pass, '-U'])
         self.master.run_command(['systemctl', 'start', 'chronyd'])
+
+
+class TestMaskMasterInstall(IntegrationTest):
+    """ Test master installation with wrong mask
+
+    This test checks that master installation fails (expectedly) if
+    mask > 022.
+
+    related ticket: https://pagure.io/freeipa/issue/7193
+    """
+
+    num_replicas = 0
+
+    @classmethod
+    def install(cls, mh):
+        pass
+
+    def test_install_master(self):
+        self.master.run_command(['umask', '0027'])
```

Comment:
The test needs to write the umask in /root/.bashrc, otherwise it won't execute as expected.
`self.master.run_command` makes a ssh connection to the master, starts a shell, calls umask then disconnects. Because of that, the umask setting is lost as soon as the `run_command` call returns.

Reply:
Thanks for the input! Change is done.